CONTRIBUTING.md

```diff
@@ -0,0 +1,63 @@
+# Contributing to the Runtime Component Operator
+
+There are multiple ways to contribute: report bugs, request features, fix bugs, contribute code, improve documentation, etc. You must follow the established process and guidelines. We welcome your contributions!
+
+- [Raising issues](#Raising-Issues)
+- [Legal](#Legal)
+- [Coding Standards](#Coding-Standards)
+
+## Raising Issues
+
+Please raise any bug reports or feature requests on the [Runtime Component Operator project repository's GitHub issue tracker](https://github.com/application-stacks/runtime-component-operator/issuess). Search the list to see if your issue has already been raised.
```

Comment:
Needs to be updated to https://github.com/application-stacks/runtime-component-operator/issues

Reply:
Good catch. Updated it. Thank you